rmf_fleet_adapter/src/rmf_fleet_adapter/load_param.cpp

```diff
@@ -88,4 +88,46 @@ rmf_traffic::agv::VehicleTraits get_traits_or_default(rclcpp::Node& node,
   return traits;
 }
 
+
+//==============================================================================
+std::optional<rmf_battery::agv::BatterySystem> get_battery_system(
+  rclcpp::Node& node,
+  const double default_voltage,
+  const double default_capacity,
+  const double default_charging_current)
+{
+  const double voltage =
+    get_parameter_or_default(node, "battery_voltage", default_voltage);
+  const double capacity =
+    get_parameter_or_default(node, "battery_capacity", default_capacity);
+  const double charging_current =
+    get_parameter_or_default(
+      node, "battery_charging_current", default_charging_current);
+
+  auto battery_system = rmf_battery::agv::BatterySystem::make(
+    voltage, capacity, charging_current);
+  
+  return battery_system;
+}
+
+//==============================================================================
+std::optional<rmf_battery::agv::MechanicalSystem> get_mechanical_system(
+  rclcpp::Node& node,
+  const double default_mass,
+  const double default_inertia,
+  const double default_friction)
+{
+    const double mass =
+    get_parameter_or_default(node, "mass", default_mass);
+  const double inertia =
+    get_parameter_or_default(node, "inertia", default_inertia);
```

Comment:
This `"inertia"` should also be changed to something more like `"moment"`.

Reply:
Renamed to `moment_of_inertia` https://github.com/osrf/rmf_core/pull/217/commits/6b661b051545cd2752059222760b1564da93004b